tests/api2/test_300_nfs.py

```diff
@@ -1734,6 +1734,35 @@ def test_manage_gids(self, start_nfs, state, expected):
             s = parse_server_config()
             assert s['mountd']['manage-gids'] == expected, str(s)
 
+    def test_rdma_config(self, start_nfs):
+        '''
+        Mock response from rdma.capable_protocols to confirm NFS over RDMA config setting
+        '''
+        assert start_nfs is True
+
+        # Confirm the setting does not exist by default
+        s = parse_server_config()
+        assert s.get('rdma') is None, str(s)
+
+        # RDMA setting should fail on a test vm.
+        with pytest.raises(ValidationErrors) as ve:
+            call("nfs.update", {"rdma": True})
+        assert ve.value.errors == [
+            ValidationError(
+                'nfs_update.rdma',
+                'This platform cannot support NFS over RDMA or is missing an RDMA capable NIC.',
+                22
+            )
+        ]
+
+        with mock("rdma.capable_protocols", return_value=['NFS']):
+            with nfs_config():
+                call("nfs.update", {"rdma": True})
```

Comment:
Turn it off again below?

Reply:
That's the point of `with nfs_config()`.  It returns the config to original.